vidarrtest-regression.json.in

```diff
@@ -2,7 +2,7 @@
     {
         "arguments": {
             "hsMetrics.reference": "hg19",
-            "hsMetrics.baitBed": "/.mounts/labs/gsi/testdata/hsMetrics/input_data/Agilent_SureSelectXT_Human_All_Exon_V5_UTRs.bed",
+            "hsMetrics.baitBed": "/.mounts/labs/gsi/testdata/hsMetrics/test_inputs/Agilent_SureSelectXT_Human_All_Exon_V5_UTRs.bed",
```

Comment:
we can change the dirname to be consistent with the workflow name (see the build file). But this may be optional